### ISSUE TITLE
some question about batch-size

### ISSUE THREAD

**Issue by txw1997** [JiechaoGuan/FSL-DAPNA#2]
Thanks for you excellent work! But I am confuse about the hyper-parameter: num_train_episodes. Is it what we always say "batch size"?

**Comment by JiechaoGuan** [comment]
Yes, almost the same. The role of 'training episode' in FSL is actually the same as that of 'mini-batch' in traditional deep learning. That is, both episode and batch are the training units. A subtle difference is that, 'batch size' means the number of training images in one batch, while 'num_train_episodes' represents the number of training episodes (each episode contains some training images).
